packages/build-utils/src/types.ts

```diff
@@ -312,13 +312,16 @@ export interface Builder {
   config?: Config;
 }
 
+export type LauncherType = 'Nodejs' | 'EdgeLight';
```

Comment:
I think `EdgeLight` is misleading since thats a runtime and not an API signature.

Perhaps `WinterCG` or `WHATWG` might be better here.

Also cc @TooTallNate @EndangeredMassa @Ethan-Arrowood who since this affects BOA.

Reply:
I've changed to `WinterCG`